components/infobox/wikis/chess/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,150 @@
+---
+-- @Liquipedia
+-- wiki=chess
+-- page=Module:Infobox/Person/Player/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Ratings = require('Module:Ratings')
+local Table = require('Module:Table')
+local TeamHistoryAuto = require('Module:TeamHistoryAuto')
+local Variables = require('Module:Variables')
+
+local Injector = Lua.import('Module:Widget/Injector')
+local Player = Lua.import('Module:Infobox/Person')
+
+local Widgets = require('Module:Widget/All')
+local Cell = Widgets.Cell
+local Title = Widgets.Title
+
+---@class ChessInfoboxPlayer: Person
+local CustomPlayer = Class.new(Player)
+local CustomInjector = Class.new(Injector)
+
+local TITLES = {
+	{code = "gm", name = "Grandmaster"},
+	{code = "im", name = "International Master"},
+	{code = "wgm", name = "Woman Grandmaster"},
+	{code = "fm", name = "FIDE Master"},
+	{code = "wim", name = "Woman International Master"},
+	{code = "cm", name = "Candidate Master"},
+	{code = "wfm", name = "Woman FIDE Master"},
+	{code = "wcm", name = "Woman Candidate Master"},
+}
+
+---@param frame Frame
+---@return Html
+function CustomPlayer.run(frame)
+	local player = CustomPlayer(frame)
+	player:setWidgetInjector(CustomInjector(player))
+
+	player.args.id = player.args.id or player.args.romanized_name or player.args.name
+
+	player.args.autoTeam = true
+	player.args.history = TeamHistoryAuto.results{
+		convertrole = true,
+		addlpdbdata = true
+	}
+
+	-- Title.
+	player.args.title = Array.find(
+		TITLES,
+		function (title, _)
+			return Logic.isNotEmpty(player.args['title_' .. title.code])
+		end
+	)
+
+	-- Rating IDs.
+	player.args.rating_id1 = player.args.rating_id or player.args.rating_id1 or player.args.fide
+	player.args.rating_ids = {}
+	for _, id, _ in Table.iter.pairsByPrefix(player.args, 'rating_id') do
+		table.insert(player.args.rating_ids, id)
+	end
```

Comment:
```suggestion
	player.args.rating_ids = Array.mapIndexes(function(index) return args['rating_id' .. index] end)
```
possibly

Reply:
When this edit is added in, this error pops (<https://liquipedia.net/chess/Module:Infobox/Person/Player/Custom/dev/h2>)
![image](https://github.com/user-attachments/assets/c450cd4a-2b76-4017-a283-cd92b3b1d05c)